package.json

```diff
@@ -78,7 +78,7 @@
     "predb": "mkdir -p db",
     "db": "mongod --dbpath db",
     "db-update": "./import-tests.sh",
-    "install-openfisca": "pip install --user --upgrade -r openfisca/requirements.txt",
+    "install-openfisca": "[ $VIRTUAL_ENV ] && pip install --upgrade -r openfisca/requirements.txt",
```

Comment:
The yielded error message in case the virtualenv is missing is:

```
› npm run install-openfisca

> sgmap-mes-aides-ui@1.0.0 install-openfisca /Users/sgmap/Dev/mes-aides-ui
> [ $VIRTUAL_ENV ] && pip install --upgrade -r openfisca/requirements.txt

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! sgmap-mes-aides-ui@1.0.0 install-openfisca: `[ $VIRTUAL_ENV ] && pip install --upgrade -r openfisca/requirements.txt`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the sgmap-mes-aides-ui@1.0.0 install-openfisca script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/sgmap/.npm/_logs/2017-05-29T14_55_28_764Z-debug.log
```

This is too unclear. If you can activate the virtualenv from the NPM scripts and make it transparent for the user, great. If not or if it's too risky, then at least provide a [clear error message](https://github.com/openfisca/openfisca-core/blob/master/CONTRIBUTING.md#great-error-messages).